lib/gds_api/test_helpers/publishing_api.rb

```diff
@@ -754,12 +754,27 @@ def stub_publishing_api_has_path_reservation_for(path, publishing_app)
                     body: { publishing_app: publishing_app, base_path: path }.to_json)
       end
 
-      def stub_publishing_api_returns_path_reservation_validation_error_for(path, error_details = nil)
-        error_details ||= { "base" => ["computer says no"] }
-        error_data = publishing_api_path_data_for(path).merge("errors" => error_details)
+      # Stub a PUT /paths/:base_path request for a particular publishing
+      # application. Calling for a different publishing application will return
+      # a 422 response.
+      #
+      # @example
+      #   stub_publishing_api_has_path_reservation_for("/foo", "content-publisher")
```

Comment:
Copy and paste error: The example doesn't match the method definition.

Reply:
Good catch - sorted.